xcode/Subconscious/Shared/Components/Detail/MemoEditorDetail.swift

```diff
@@ -1423,24 +1441,13 @@ struct MemoEditorDetailModel: ModelProtocol {
         detail: MemoEditorDetailResponse,
         autofocus: Bool
     ) -> Update<MemoEditorDetailModel> {
-        guard autofocus else {
-            // If autofocus is false, request blur if needed
-            return update(
-                state: state,
-                actions: [
-                    .setDetailLastWriteWins(detail),
-                    .requestEditorFocus(false)
-                ],
-                environment: environment
-            )
-        }
```

Comment:
Intentional removal?

Reply:
Not fully intentional, I forgot to restore this code. It was behaving from a UX POV but I've clarified the code to explode the thinking behind it.